src/components/Widgets/WidgetBase.vue

```diff
@@ -603,6 +155,15 @@ export default {
     hideControls() {
       return this.widget.hideControls;
     },
+    component() {
+      const type = COMPAT[this.widgetType] || this.widget.type;
+      if (!type) {
+        ErrorHandler('Widget type was not found');
+        return null;
+      }
+      // eslint-disable-next-line prefer-template
+      return () => import('@/components/Widgets/' + type + '.vue');
```

Comment:
If the specified widget type isn't found, then we should show an error message

Reply:
I wasn't sure of the best method to do this. If you check the known ```COMPAT``` object, then that means no widgets can be loaded that are not on that map, eliminating the ability to pass a custom type. We could check the filesystem but that doesn't seem ideal either. As it stands now, the page will load, that section will appear empty and there is an error in the console.

![Screen Shot 2022-06-24 at 7 34 56 AM](https://user-images.githubusercontent.com/1407228/175558313-80a9e857-1c4c-4a93-a752-c8af0dd74eb7.png)

Thinking about it some more this morning, I'll catch the import error, log it to the console, and render the Blank widget instead. I should have another PR ready to go in a bit.